practice/rain.py

```diff
@@ -1,86 +1,214 @@
 """ Opens a Rain Text File and Mines for information. I'm using "cells" for rows, and "boxes" for columns"""
```

Comment:
It looks like you have some of the Rain problem committed in the `master` branch. In the future, use the fact that your Git branch is now in your shell prompt to ensure that you're only committing to the branch for the practice problem we're on.